pygmt/tests/test_figure.py

```diff
@@ -373,12 +373,31 @@ def test_figure_display_external():
     fig.show(method="external")
 
 
-def test_figure_set_display_invalid():
+class TestSetDisplay:
```

Comment:
We don't often use classes for the pytest unit tests, is there a reason to use it here?

Reply:
Just want to group multiple tests so that it is easier to find all tests for testing a specific function.

Reply:
Ok, it looks like we're using `TestGetDefaultDisplayMethod` below, so ok with this.